content/plugins/aws.md

```diff
@@ -10,7 +10,10 @@ weight: 100
 
 # Description
 
-The AWS plugin allows users to use Cloudify to manage cloud resources on AWS.
+The AWS plugin allows users to use Cloudify to manage cloud resources on AWS. See below for currently supported resource types.
+
+Be aware that some services and resources vary in availability between regions and accounts.
+
 For more information about the library, please refer to: [http://boto.readthedocs.org/en/latest/index.html](http://boto.readthedocs.org/en/latest/index.html).
```

Comment:
For more information about boto, please refer [here](...). 

No need to show the link itself.